service/skills_service.py

```diff
@@ -0,0 +1,19 @@
+from dal.skills_dal import SkillsDAL
```

Comment:
Maybe the service is not required when you have the DAL present. This is my opinion though, and not a hard rule.